.github/workflows/release.yml

```diff
@@ -1,43 +1,9 @@
-name: "Module Release"
+name: "Publish module"
 
 on:
   workflow_dispatch:
 
 jobs:
-  release:
-    name: "Release"
-    runs-on: "ubuntu-latest"
-    if: github.repository_owner == 'puppetlabs'
-
-    steps:
-
-      - name: "Checkout"
-        uses: "actions/checkout@v3"
-        with:
-          ref: "${{ github.ref }}"
-          clean: true
-          fetch-depth: 0
-
-      - name: "Get version"
-        id: "get_version"
-        run: |
-          echo "version=$(jq --raw-output .version metadata.json)" >> $GITHUB_OUTPUT
-
-      - name: "PDK build"
-        uses: "docker://puppet/pdk:nightly"
-        with:
-          args: "build"
-
-      - name: "Generate release notes"
-        run: |
-          export GH_HOST=github.com
-          gh extension install chelnak/gh-changelog
-          gh changelog get --latest > OUTPUT.md
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: "Create release"
-        run: |
-          gh release create v${{ steps.get_version.outputs.version }} --title v${{ steps.get_version.outputs.version }} -F OUTPUT.md
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  release: 
+    uses: "puppetlabs/cat-github-actions/.github/workflows/module_release.yml@main"
+    secrets: "inherit"
```

Comment:
this module can't use this reusable workflow.
The module is not on the forge so will fail.